handlers/main.yml

```diff
@@ -0,0 +1 @@
+# Handlers for passwd
```

Comment:
With no expectation of ever using this, we can delete the whole `handlers` directory.